Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -547,10 +548,9 @@ class ITinyMCEPluginSchema(Interface):
             SimpleTerm('visualchars', 'visualchars', u"visualchars"),
             SimpleTerm('wordcount', 'wordcount', u"wordcount")
         ])),
-        default=['advlist', 'directionality', 'emoticons',
-                 'fullscreen', 'hr', 'insertdatetime', 'lists', 'media',
+        default=['advlist', 'fullscreen', 'hr', 'lists', 'media',
                  'nonbreaking', 'noneditable', 'pagebreak', 'paste', 'preview',
-                 'print', 'save', 'searchreplace', 'tabfocus', 'table',
+                 'print', 'save', 'tabfocus', 'table',
```

Comment:
No, this was on purpose. There is no integration with "save" so it was removed. See note above new "save" button.